packages/compendium-cli/utils/appsync.go

```diff
@@ -47,6 +50,7 @@ func RunAppSyncQuery(ctx context.Context, client *appsync.Client, query string,
 
 	err = res.DataAs(response)
 	if err != nil {
+		fmt.Println(err)
```

Comment:
Out of curiosity. Why are we adding these logs? The final error should "bubble" to the actual put command which will return it as exit error with given description.

Reply:
hmmm,  as i was debugging this (with typo in graphql mutation) it was very hard to figure out where error occured and since we need to "deploy and try" i think having every single possible logging location is desirable. I m not really proficient in go but 
```go
err = callWithError()
if err != nil {
    return
}
```
is equivalent to
```typescript
try {
    callWithError();
} catch (err) {
    return;
}
```

maybe i m missing something but go's errors are values so no bubbling should occur right? if we want to have error in caller function, we need to return it as a value

Reply:
Actually it's:

```
try {
    callWithError();
} catch (err) {
    return;
}
```
as the `err` is defined on line 30 and assigned on line 31 :-) 
Check out [named return values](https://go.dev/tour/basics/7)

Reply:
damn totally forgot about this feature (never liked it tbh :D) ...ok so maybe there was some other issue, it just wasnt clear from the error where its coming from. But agreed that this wont help it either, will try to think it through on next task :)